web-client/src/app/views/search-wallet/search-wallet.page.html

```diff
@@ -0,0 +1,68 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-back-button></ion-back-button>
+    </ion-buttons>
+  </ion-toolbar>
+  <ion-toolbar>
+    <div class="ion-text-center p-2">
+      <h1 class="font-semibold font-nasalization">Search wallet address</h1>
+    </div>
+  </ion-toolbar>
+</ion-header>
+
+<ion-content>
+  <ion-grid>
+    <ion-row>
+      <ion-col>
+        <ion-list lines="none" class="form-item">
+          <ion-item>
+            <ion-input placeholder="Name" [(ngModel)]="name"></ion-input>
+          </ion-item>
+          <ion-item>
+            <ion-input
+              placeholder="Phone Number"
```

Comment:
The name and phone number must be exactly right, including the plus sign before the country code. I could only get it to work by copying and pasting from the database. We could implement a validation where it doesn't let you search unless the number is the right format. Something easier might be just to add to the form cell placeholder with the format. 

I.e. instead of "Phone Number", maybe "Phone Number (+05555555555)".

Reply:
Thanks for the suggestion. Would the below changes suffice?
![image](https://user-images.githubusercontent.com/68713894/201746485-d13f9a68-60bf-4177-8f48-fc30c582bbb6.png)

Reply:
Yes that is better. We could also reuse the same form validation that is used for phone numbers during the create wallet process.

However, I think we need to add validation to several other places, so maybe we can do that as a separate task.